### DISCUSSION TITLE
BBCode editor that supports live view instead of using the preview button and coding again on errors.

### DISCUSSION THREAD

**Discussion by giyugg** [ppy/osu#26418] [Ideas] [answered]
Saw a reddit post on this topic. 
https://www.reddit.com/r/osugame/comments/17jhhwn/looking_for_a_bbcode_editor_that_supports/

I was hoping to see that osu! will provide this live view for bbcode so that when we post forum or editing our "me!", we don't need to rely on clicking the preview button again and again.

**Comment by bdach** [comment, marked as answer]
proper place for this is https://github.com/ppy/osu-web/issues/10845
